### PR TITLE
Fix Helloworld golang sample

### PR DESCRIPTION
**Pull request by akashrv**
Fixes [#3011](https://github.com/knative/eventing/issues/3011)
## Proposed Changes 
- Updated to use go 1.13
- Updated to use go mod
- Fixed the default broker URL

### CODE REVIEW

File: docs/eventing/samples/helloworld/helloworld-go/README.md [hunk 1]
```diff
@@ -277,7 +281,9 @@ Helloworld-go app logs the context and the msg of the above event, and replies
 back with another event.
 
 1.  Display helloworld-go app logs
-    `shell kubectl --namespace knative-samples logs -l app=helloworld-go --tail=50`
+    ```shell
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-go/README.md:284]: Remove trailing whitespace:
```suggestion
    ```shell
```

File: docs/eventing/samples/helloworld/helloworld-go/README.md [hunk 2]
```diff
@@ -277,7 +281,9 @@ Helloworld-go app logs the context and the msg of the above event, and replies
 back with another event.
 
 1.  Display helloworld-go app logs
-    `shell kubectl --namespace knative-samples logs -l app=helloworld-go --tail=50`
+    ```shell 
+    kubectl --namespace knative-samples logs -l app=helloworld-go --tail=50
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-go/README.md:285]: Remove trailing whitespace:
```suggestion
    kubectl --namespace knative-samples logs -l app=helloworld-go --tail=50
```

File: docs/eventing/samples/helloworld/helloworld-go/Dockerfile [hunk 3]
```diff
@@ -1,30 +1,30 @@
 # Use the offical Golang image to create a build artifact.
 # This is based on Debian and sets the GOPATH to /go.
 # https://hub.docker.com/_/golang
-FROM golang:1.12 as builder
+FROM golang:1.13 as builder
```
Comment (vaikas) [inline docs/eventing/samples/helloworld/helloworld-go/Dockerfile:4]: do you just want to move to 1.14 while at it?

File: docs/eventing/samples/helloworld/helloworld-go/go.mod [hunk 4]
```diff
@@ -0,0 +1,8 @@
+module knative.dev/docs/docs/eventing/samples/helloworld/helloworld-go
+
+go 1.13
```
Comment (vaikas) [inline docs/eventing/samples/helloworld/helloworld-go/go.mod:3]: and here.

**Pull request merged**